packages/beeq/src/components/dropdown/bq-dropdown.tsx

```diff
@@ -69,6 +69,14 @@ export class BqDropdown {
     this.bqOpen.emit({ open: this.open });
   }
 
+  @Watch('disabled')
+  handleDisabledChange(newValue: boolean) {
+    const bqButton = document.querySelector('bq-button');
+    if (!bqButton) return;
+
+    bqButton.disabled = newValue;
+  }
+
```

Comment:
Considering that we already have a `triggerElem` being set in the line:92,  maybe we could:

```ts
  @Watch('disabled')
  handleDisabledChange(newValue: boolean) {
    if (!this.triggerElem) return;
    // Optional: check if the triggerElem is a bq-button
    // if(!isHTMLElement(this.triggerElem, 'bq-button')) return;

    this.triggerElem?.setAttribute('disabled', 'true');
  }
```

cc. @endv-bogdanb